### DISCUSSION TITLE
Noise Harmonics

### DISCUSSION THREAD

**Discussion by andrejhronco** [cables-gl/cables_docs#134] [Feature Requests] [answered]
Love using the various noise ops in cables (really looking forward to getting texture offset) and I was playing around with the Noise Op in TouchDesigner and there's a really nice param called Harmonics : "The number of higher frequency components to layer on top of the base frequency." It adds a rich texture to the noise. would love to see this in cables.
Here's a video that demonstrates the feature: https://youtu.be/DzD2ilLO-9Q?t=281

**Comment by pandrr** [comment, marked as answer]
thanks for the suggestion.

on dev.cables.gl harmonics are implemented in cellularNoise and perlinNoise for now

**Reply by andrejhronco** [reply]
Awesome! looks great. thank you! is the harmonic limit of ints 1 - 5 a practicality? how would a larger range with floats behave?
